conf/modules.config

```diff
@@ -193,23 +193,33 @@ process {
     }
 
 // TRIMMING
-    withName: 'TRIMGALORE' {
-        ext.args         = '--fastqc'
-        publishDir       = [
-            [
-                path: { "${params.outdir}/reports/trimgalore/${meta.id}/fastqc" },
-                mode: params.publish_dir_mode,
-                pattern: "*{html,zip}"
-            ],
+
+    withName: FASTP {
+        ext.args = [ "",
+            params.trim_fastq              ?: "--disable_adapter_trimming",
+            params.clip_r1 > 0             ? "--trim_front1 ${params.clip_r1}" : "", // Remove bp from the 5' end of read 1.
+            params.three_prime_clip_r1 > 0 ? "--trim_tail1 ${params.three_prime_clip_r1}" : "", // Remove bp from the 3' end of read 1 AFTER adapter/quality trimming has been performed.
+            params.clip_r2   > 0           ? "--trim_front2 ${params.clip_r2}" : "", //Remove bp from the 5' end of read 2.
+            params.three_prime_clip_r2 > 0 ? "--trim_tail2 ${params.three_prime_clip_r2}" : "", // Remove bp from the 3' end of read 2 AFTER adapter/quality trimming has been performed.
+            params.trim_nextseq            ? "--trim_poly_g" : "", // Apply the --nextseq=X option, to trim based on quality after removing poly-G tails.
+            params.split_fastq > 0         ? "--split_by_lines ${params.split_fastq * 4}" : "",
+
+            // For trimgalore also auto detection was used
+            //params.adapter_R1       ? "--adapter_sequence ${params.adapter_R1}" : "", //normally not needed
```

Comment:
should we provide the additonal parameter for setting specific adapter sequences? Wasn't supported before, but trimgalore guessed the adapter (same here at the moment by default) @maxulysse